### ISSUE TITLE
One voice for Mirage

### ISSUE THREAD

**Issue by andrewlotel** [qwbarch/mirage#117]
Hello. So, apparently, whenever my friends and I play LC with Mirage, mobs only mimic the voice of the host, who I am. Masked-s of me work perfectly fine and speak my voice, and all the mobs speak only my voice. If there is a masked of other player, he just wanders silently. Is it a modpack issue or a recent update issue? Thank you in adanvce.

Just in case, pinning down modpack code: 0193cb6e-5988-485d-8119-fe7f7934b943

**Comment by qwbarch** [comment]
Can you ask your friend to check their Lethal Company folder for thr
Mirage/Recording folder, and see if any .opus files are being created?

I think the issue is that for whatevet reason, recordings aren't being
created for some players

**Comment by qwbarch** [comment]
Also, can you ask if your friend has a windows username that contains
accents in it? I'm wondering if this is a unicode problem with the file
path again

**Comment by andrewlotel** [comment]
Alright, it's the modpack issue. On our main modpack the recordings folder is created only if you're hosting, and if you're connecting, there is no folder. We created a new modpack with solely Mirage and Imperium, and voila! Sorry for the trouble, modding is a pain in the neck. Gonna look into it on our own now. And thank you for the great mod!

**Comment by qwbarch** [comment]
Do you mind letting me know if you do find the issue?
Sounds like a mod conflict if it's a modpack issue, so I'm wondering which
mod is causing the conflict with Mirage

**Comment by andrewlotel** [comment]
To be honest, I still have no clue what the issue was. We remade the modpack and it didn't work. Then we started turning mods on one by one and launching game to make sure that everything works. We hoped we would find the impostor, but when we finally turned everything on, it was working perfectly fine. I think it has something to do with the order of installation, and I don't know which mod exactly caused all the trouble.
